File: 0782-jewels-and-stones/0782-jewels-and-stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int result = 0;
        for (char stone: stones) {
            for (char jewel: jewels) {
                if (stone == jewel) result++;
            }
        }
        return result;
    }
};

File: 0782-jewels-and-stones/0782-jewels-and-stones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        int result = 0;
        
        for (char stone : stones.toCharArray()) {
            if (jewels.indexOf(stone) != -1) {
                result++;
            }
        }

        return result;
    }
}